protocol/0066-VALW-validator_tendermint_weights.md

```diff
@@ -15,37 +15,67 @@ If the network has no stake at all, then all validators would have equal voting
 
 ### Basic sanity check (<a name="0066-VALW-001" href="#0066-VALW-001">0066-VALW-001</a>)
 1. set up a network with 5 validators 
-1. give each of the validators the following number of self-staked tokens: `2000`
-1. wait for the delegation to become active in the next epoch
-1. when the epoch begins verify that the delegation went through
-1. check that Tendermint weights for the validators are `2000`
```

Comment:
I don't follow why we're removing the weights. The point is that 5 validators with the same amount of tokens staked and delegated should have the same weight. The weight is an integer (out of 10 000).

Reply:
if you add the expected weights I can verify against the results we see. Currently all results are calculated based on rankingScore values and values from tendermint and these are used to verify results.

Reply:
> Currently all results are calculated based on rankingScore values and values from tendermint and these are used to verify results.

Then let's add the formula here and say that the formula give the right answer.

Reply:
@vega-paul - please can you update the AC and add the formulas and numbers. We can then test this, review the numbers and decide the next steps?